packages/strapi-design-system/src/Link/Link.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { TableLabel } from '../Text';
+import { Box } from '../Box';
+
+const LinkWrapper = styled.a`
+  text-transform: uppercase;
+  text-decoration: none;
+  pointer-events: ${({ disabled }) => (disabled ? 'none' : undefined)};
+
+  svg path {
+    fill: ${({ disabled, theme }) => (disabled ? theme.colors.neutral600 : theme.colors.primary600)};
+  }
+
+  svg {
+    font-size: ${10 / 16}rem;
+  }
+
+  &:hover,
+  &:active {
+    color: ${({ theme }) => theme.colors.primary800};
+  }
+`;
+
+// TODO: make sure to use the link from the router library chosen
+export const Link = ({ href, to, children, disabled, leftIcon, rightIcon, ...props }) => {
```

Comment:
The href will be used for external links ?

Reply:
Yes